custom-fonts.md

```diff
@@ -23,6 +41,83 @@ flutter:
         - asset: assets/fonts/abrilfatface/AbrilFatface-Regular.ttf
 ```
 
+### Second use the font
+
+Second use the font by delcaring the [`TextStyle`](https://docs.flutter.io/flutter/painting/TextStyle-class.html)
+constant and adding the font family used in the `pubsec.yaml`.
+
+```
+// declare the text style
+const textStyle = const TextStyle(
+  fontFamily: 'Raleway',
+);
+
+// use the text style
+var buttonText = const Text(
+  "Use the font for this text",
+  style: textStyle,
+);
+```
+
+## Using the Included Material Design fonts
+If you're looking at using the Material Design fonts,
+they can be simply included by adding a property `uses-material-design: true` to the `pubspec.yaml` file.
+
+```
+flutter:
+  # The following line ensures that the Material Icons font is
+  # included with your application, so that you can use the icons in
+  # the Icons class.
+  uses-material-design: true
+```
+
+### Included Material Design fonts
+Here are the included fonts that come with `uses-material-design: true`.
+
+| Font | Features |
+| ---- | -------- |
+| Roboto-Regular.ttf | |
+| Roboto-Italic.ttf | italic |
+| Roboto-Thin.ttf | 100 |
+| Roboto-ThinItalic.ttf | 100, italic |
+| Roboto-Light.ttf | 300 |
+| Roboto-LightItalic.ttf | 300, italic |
+| Roboto-Medium.ttf | 500 |
+| Roboto-MediumItalic.ttf | 500, italic |
+| Roboto-Bold.ttf | 700 |
+| Roboto-BoldItalic.ttf | 700, italic |
+| Roboto-Black.ttf | 900 |
+| Roboto-BlackItalic.ttf | 900, italic |
+| &nbsp; | &nbsp; |
+| RobotoCondensed-Regular.ttf | |
+| RobotoCondensed-Italic.ttf | italic |
+| RobotoCondensed-Light.ttf | 300 |
+| RobotoCondensed-LightItalic.ttf | 300, italic |
+| RobotoCondensed-Bold.ttf | 700 |
+| RobotoCondensed-BoldItalic.ttf | 700, italic |
+
+The included fonts
+[material_fonts.yaml](https://github.com/flutter/flutter/blob/master/packages/flutter_tools/schema/material_fonts.yaml)
+reference file.
+
+### Declaring in the font asset
+This is an example using the Material Design fonts.
```

Comment:
I sort of think I need to make a reference to the default theme font, but I don't have enough time to figure it out.